src/stories/common/GlobalSearchBar.stories.tsx

```diff
@@ -5,6 +5,7 @@ import {GlobalSearchBar} from '@/components/common/search/GlobalSearchBar';
 const meta: Meta<typeof GlobalSearchBar> = {
   title: 'Components/Common/Global SearchBar',
```

Comment:
```suggestion
  title: 'Components/Common/Search/Global Search Bar',
```

스토리 폴더 구조 컴포넌트 구조랑 일치시켜 주세용